src/commands/record.rs

```diff
@@ -43,22 +45,29 @@ fn write_asciicast_event<W: ?Sized>(
     event_type: asciicast::EventType,
     since_start: Duration,
     data: &[u8],
+    raw: bool,
```

Comment:
Rather than having this as an option, let's make another function like `write_raw_event`. This will make it so we can more easily unit test each path. I'll likely eventually move everything into some sort of Writer trait with multiple implementors.